src/instructlab/model/backends/backends.py

```diff
@@ -255,26 +255,37 @@ def ensure_server(
         if backend == VLLM:
             # TODO: resolve how the hostname is getting passed around the class and this function
             vllm_server_process = server_process_func(port, background)
-            logger.info(f"Starting a temporary vLLM server at {temp_api_base}")
+            logger.info("Starting a temporary vLLM server at %s", temp_api_base)
             count = 0
             # TODO should this be configurable?
-            vllm_startup_timeout = 60
-            while count < vllm_startup_timeout:
-                if check_api_base(temp_api_base, http_client):
-                    logger.info(f"vLLM engine successfully started at {temp_api_base}")
-                    break
+
+            vllm_startup_max_attempts = 80  # Each call to check_api_base takes >2s + 2s sleep = ~5 mins of wait time
```

Comment:
I think we should keep the TODO to make this configurable eventually, with `80` serving as a default value

Reply:
The TODO is still above right?

Reply:
Apologies, yes it is, misread the change